flaml/model.py

```diff
@@ -514,7 +500,7 @@ def _set_training_args(self, **kwargs):
 
         self._training_args.eval_steps = (
             self._training_args.logging_steps
-        ) = self._training_args.saving_steps = self.checkpoint_freq
+        ) = self._training_args.saving_steps = 500  # self.checkpoint_freq

```

Comment:
why fixing it as a constant?

Reply:
So the evaluation frequency is not too high.

Reply:
my question is why fixing it a constant.
The original code uses `self._checkpoint_freq`. If you keep using `self._checkpoint_freq`, can't you make "evaluation frequency not too high"?

Reply:
I want to propose to remove self._checkpoint_freq, it’s less interpretable than eval_steps setting as a constant.

Reply:
Then remove it and write it down. Right now it's not removed and your proposal is not written down in the PR.